needrestart

```diff
@@ -1397,6 +1411,37 @@ if($opt_p) {
 
     exit $ret;
 }
+if ($opt_o) {
+    print "# HELP needrestart_build_info information about needrestart's runtime build\n";
+    print "# TYPE needrestart_build_info info\n";
+    print "needrestart_build_info{version=$NeedRestart::VERSION,perl_version=$^V} 1\n";
+
+    my $ometric_now = time();
+    print "# HELP needrestart_timestamp when, in unix timestamp, needrestart was last updated\n";
+    print "# TYPE needrestart_timestamp gauge\n";
+    print "needrestart_timestamp $ometric_now\n";
+
+    if ($opt_k) {
+        print "# HELP needrestart_kernel_info information about the kernel\n";
+        print "# TYPE needrestart_kernel_info info\n";
+        print "needrestart_kernel_info{running=$ometric_kernel_values{krunning},expected=$ometric_kernel_values{kexpected}} $ometric_kernel_values{kstatus}\n";
```

Comment:
no currently it will look like `needrestart_kernel_info{running=6.9.7,expected=6.9.7} current` on success

and `needrestart_kernel_info{running=6.9.6,expected=6.9.7} obsolete` on failure.


that detail was something that I was really not sure about how to output.. outputting a binary value could be interesting in the case of the kernel since there's currently only two possible states. however, for microcode, there are three different states, `unknown`, `current` and `obsolete`. so to keep things consistent, I decided to output the state "name" for both.

should the kernel_info metric be a bool instead?